integrations/access/pagerduty/pagerduty_test.go

```diff
@@ -596,7 +596,12 @@ func (s *PagerdutySuite) assertNewEvent(watcher types.Watcher, opType types.OpTy
 			assert.Equal(t, resourceKind, ev.Resource.GetKind())
 			assert.Equal(t, resourceName, ev.Resource.GetName())
 		} else {
-			assert.Nil(t, ev.Resource)
```

Comment:
Teleport >= 13 sends a `*WatchStatusV1` in `OpInit`. We previously ran tests with 12.x, so we did not hit this.

Reply:
Yeah, I saw that too when upgrading the teleport in teleport-plugins to v13